README.md

```diff
@@ -58,3 +147,7 @@ const scenarioNameFinder = new GherkinDocumentWalker({}, {
   handleScenario: (scenario) => allScenarioNames.push(scenario.name),
 })
 ```
+
+## Support
+
+Support is [available from the community](https://cucumber.io/tools/cucumber-open/support/) if you need it.
```

Comment:
I would think it better to leave this out. Support is only available in the sense that people might volunteer to answer questions.